tests/unit/plugins/module_utils/mock_maskingview_api.py

```diff
@@ -74,3 +89,33 @@ def get_create_mv_exception_response(response_type):
             return "Host test_host_2 does not exist"
         elif response_type == 'hostgroup':
             return "Host Group test_host_gp_1 does not exist"
+
+    @staticmethod
+    def get_create_mv_create_exception_response(response_id):
+        return ('Create masking view %s failed; error ' % response_id)
```

Comment:
Another future improvement 
>For Python 3.6 and later, f-strings are the preferred method due to their readability and performance. For earlier versions, the str.format() method is a suitable alternative. It's advisable to avoid the % operator for string formatting in new codebases.